components/atoms/Button.tsx

```diff
@@ -0,0 +1,23 @@
+import { styled } from '@/stitches.config';
+import { PropsWithChildren } from 'react';
+
+export interface ButtonProps extends PropsWithChildren {}
```

Comment:
What is the purpose of this interface?

Reply:
I had declared that interface during the initial development stages of the project as a placeholder for the prop types before I went and implemented the component using the Stitches `styled()` function. Forgot to clean that up 😜